cluster/gce/config-default.sh

```diff
@@ -36,6 +36,11 @@ MASTER_ROOT_DISK_SIZE=${MASTER_ROOT_DISK_SIZE:-$(get-master-root-disk-size)}
 NODE_DISK_TYPE=${NODE_DISK_TYPE:-pd-standard}
 NODE_DISK_SIZE=${NODE_DISK_SIZE:-100GB}
 NODE_LOCAL_SSDS=${NODE_LOCAL_SSDS:-0}
+# An extension to local SSDs allowing users to specify block/fs and SCSI/NVMe devices
+# Format of this variable will be "#,scsi/nvme,block/fs" you can specify multiple
+# configurations by seperating them by a semi-colon ex. "2,scsi,fs;1,nvme,block"
+# is a request for 2 SCSI formatted and mounted SSDs and 1 NVMe block device SSD.
+NODE_LOCAL_SSDS_EXT=${NODE_LOCAL_SSDS_EXT:-""}
```

Comment:
I wonder if it's because on gce it's defaulted to "", but on GKE it's unset.

Reply:
Yeah I found this error in the master logs:

```
/home/kubernetes/bin/configure-helper.sh: line 98: NODE_LOCAL_SSDS_EXT: unbound variable
```

We keep running into this every time a new var is added, because GKE does not use `config-default.sh` at all, and `configure-helper.sh` uses `nounset`.

Reply:
@enisoc how has this been fixed in the past? I was just going to add a 'NODE_LOCAL_SSDS_EXT=${NODE_LOCAL_SSDS_EXT:-""}' to configure-helper.sh but that feels like maybe there might be a better solution